azkaban-web-server/src/main/java/azkaban/scheduler/QuartzScheduler.java

```diff
@@ -72,8 +73,36 @@ public QuartzScheduler(final Props azProps) throws SchedulerException {
   }
 
   public void start() throws SchedulerException {
-    this.scheduler.start();
-    logger.info("Quartz Scheduler started.");
+    try {
+      this.scheduler.start();
+      logger.info("Quartz Scheduler started.");
+    } catch (SchedulerConfigException e) {
+      if(e.getCause() instanceof org.quartz.JobPersistenceException) {
+        if(e.getCause().getCause() instanceof java.io.InvalidClassException) {
+          if(enableSerializationHack()) {
+            try {
+              this.scheduler.start();
```

Comment:
Are you planning to change old serialized objects in db? or you will follow same way and keep multiple versions of serialized objects? I personally am not comfortable with catching exception and add then constructing old version objects.

Reply:
If the object is ever persisted again, it will have the new serialization id as that is a static value. To iterate through and overwrite all objects would be significantly more functionality (and thus more possibility of something going wrong.) I am attempting to provide a minimally invasive solution.

Reply:
@acharneski Thank you so much for working on this PR.